### ISSUE TITLE
Diagnostic Settings for Event Hub

### ISSUE THREAD

**Issue by grik001** [Azure/logicapps#72]
Hi,

I have set up a test Logic App within an ISE inside a VNET. When I enable diagnostic settings over EventHub, for some reason no messages are received unless I disable the VNET network restrictions and allow all networks. To test this out I have done the below test:

1) Disabled function reading from EH(reading LA diagnostic info)
2) Switched Network security for EH to ALL
3) Generated Load on LA
4) Switched Network security for EH to VNET only
5) Started function, data is read successfully and logged

If I do not set the EH security to ALL no data is ever logged, the reason I did this is to prove that the function is actually working when data is present.

Are there any open issues with ISE within VNET and Diagnostic settings via EH?

EDIT:

So basically now I did another test. I have created another LA within the ISE with an action to post to EventHub(using the same hub that diag settings are pointing to). I have used the same schema(since my function is processing that) that is received from the Diagnostic settings. This worked, so basically, data is making it to the EH from the LA(inside ISE) and the Azure Function(inside ASE) is reading that Event, but for some reason, the diagnostic settings never make it to the EH.

![image](https://user-images.githubusercontent.com/27157850/61992892-8ff61c80-b064-11e9-8c1c-62708117c671.png)

**Comment by joechung-msft** [comment]
If you have questions about Logic Apps, please visit the [MSDN forums](https://social.msdn.microsoft.com/Forums/en-US/home?forum=azurelogicapps).  The Issues section here is for questions about the artifacts in this GitHub repository.
